packages/common/test/core/schemas/internal/filterSchemaToUiSchema.test.ts

```diff
@@ -31,8 +31,7 @@ describe("filterSchemaToUiSchema util:", () => {
           type: "Section",
           labelKey: "section1.label.key",
           options: {
-            section: "stepper",
-            open: true,
+            section: "stepper"
           },
```

Comment:
Assume everything's fine that we lost the "open" here?

Reply:
yes, this does not impact the test and open isn't actually an option for stepper sections, so just removing it from the test for consistency.